src/ActionList/Item.tsx

```diff
@@ -266,27 +243,25 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
         >
           <ItemWrapper {...wrapperProps}>
             <Selection selected={selected} />
-            {
-              // If we're showing an inactive indicator and a leading visual has been passed,
-              // replace the leading visual with the inactive indicator.
-              //
-              // Inactive items without a leading visual place the inactive indicator in the
-              // trailing visual slot. This preserves the left alignment of item text.
-              showInactiveIndicator && slots.leadingVisual ? (
-                // using a non-null assertion for `inactiveText` since we check for it in `showInactiveIndicator`
-                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                <InactiveIndicator labelId={labelId} text={inactiveText!} visualComponent={LeadingVisual} />
-              ) : (
-                // If it's not inactive, just render the leading visual slot
-                slots.leadingVisual
-              )
-            }
+            <VisualOrIndicator
```

Comment:
When I test this with VoiceOver, action list item and the loading indicator are announce as in totally separate things. I am not sure what would be the solution here, sorry I can't provide more in that regard however I am struggling to understand the connection between the item and its state. 

In [ToggleSwitch](https://github.com/primer/react/pull/3902), the toggle button is described by the loading text when it is in the loading state. I wonder if we need to make such connection here? 🤔

Reply:
> When I tested this with VoiceOver, action list item and the loading indicator are announce as totally separate things.

I think this is ok. Let's see what a11y has to say. @alexislucio is this week's a11y FR.

Reply:
I think this example is different due to the `Spinner` changes being moved, but I was wondering if there'd be any point in time where there could be more than one item in a loading state? I think if multiple items could be loading at once, we'd want to make the text unique as to convey what exactly is loading.

If only one item can be loading at a given time, then I think there should be enough context on what is loading due to the user triggering the thing that puts it in a loading state.

Reply:
We discourage showing too many loading indicators on the page at once, but it's certainly possible. I think the most common example (an ActionList with multiple items in a loading state) will be covered by the loading ActionList PR.

If you think we need another example of multiple Spinner loading indicators on the page at once, I'd love some ideas for use cases we can show in Storybook.

Reply:
Yes, it's possible that multiple items could be loading at once. If we can't even know the title of the list item, it will be difficult to come up with unique text for each loading item. Any ideas how we can convey that? My first thought is "Loading item {n}" or "Loading item {n} under the {section title} section".